action.yml

```diff
@@ -33,32 +33,17 @@ runs:
         repository: ${{ inputs.full_repository }}
         ref: ${{ github.event.inputs.head_ref }}
         fetch-depth: 0
-        path: "usercode/"
+        path: "headbranch/"
     - name: Create GitStream folder
       id: create-gitstream-folder
       shell: bash
       run: |
         mkdir gitstream
-    - name: Get Diff Base
-      id: get-diff
+    - name: copy usercode to gitstream folder
+      id: copy-user-code-to-gitstream-fodler
       shell: bash
       run: |
```

Comment:
Why do you need this step? i mean you could either set the path on the sep before to `gitstream/headbranch` 
Do you need 2 copies of this dir? you end up with 2 source dirs: `headbranch` and `gitstream/headbranch`

Reply:
you are right, i don't need it